### PR TITLE
Remove print statements

### PR DESCRIPTION
**Pull request by slesaad**
- Removed stdout when importing the package
- Updated the version
- Updated the changelog

### CODE REVIEW

**Comment by xhagrg** [comment]
👍 . For future: We need to think about how we can let the users know we are using the local copy instead of the remote copy.

File: pyQuARC/code/gcmd_validator.py [hunk 1]
```diff
@@ -83,15 +83,16 @@ def _download_files(force=False):
         if gcmd_date.date() < current_datetime.date() or force:
             try:
                 for keyword, link in GCMD_LINKS.items():
-                    print(f"Downloading {keyword} file from GCMD ...")
-                    response = urllib.request.urlopen(GCMD_LINKS[keyword])
+                    # Downloading updated gcmd keyword files
+                    response = urllib.request.urlopen(link)
```
Comment (xhagrg) [inline pyQuARC/code/gcmd_validator.py:87]: Why not normal requests? why use urllib?

Comment (slesaad) [reply pyQuARC/code/gcmd_validator.py]: i don't remember lol, i've updated it to use requests

**Comment by slesaad** [comment]
@xhagrg ok to merge?

**Review by xhagrg — APPROVED** [review]
LGTM

**Pull request merged**